projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-results/search-ai-results.component.ts

```diff
@@ -108,17 +120,19 @@ export class SearchAiResultsComponent extends PageComponent implements OnInit, O
     private thumbnailService: ThumbnailService,
     private nodesApiService: NodesApiService,
     private userPreferencesService: UserPreferencesService,
-    private translateService: TranslateService
+    private translateService: TranslateService,
+    protected searchAiService: SearchAiService
```

Comment:
You don't need this service here. This service is already added to parent's class (PageComponent) and it's protected so you should be able to access this here as it's child class.